wurst/systems/spells/JumpSystem.wurst

```diff
@@ -0,0 +1,112 @@
+package JumpSystem
+import ChatCommands
+import GameConfig
+import ClosureForGroups
+import ClosureTimers
+
+constant let DEFAULT_GRAVITY = 175.
+constant let STORMCROWFORM_ID = 'Amrf'
+
+init
+    registerCommandAll("jump") (triggerPlayer, args) ->
+        if (gameConfig.isTestModeEnabled() == true)
+            forUnitsSelected(triggerPlayer) u ->
+                if (u.isType(UNIT_TYPE_HERO))
+                    thiefJump(u, u.getPos().polarOffset(angle(0), 400))
+        else
+            printTimedToPlayer("Requires test mode!", 5., triggerPlayer)
+        
+public function thiefJump(unit u, vec2 targetPos)
+    let distance = u.getPos().distanceTo(targetPos)
+    let groundSpeed = 1.5 * distance
+    let gravity = 175.
+    new JumpInstance(u, targetPos)
+
+    //Play backflip animation
+    u.pause()
+    u.setAnimation(2)
+    //Skip to the actual backflip portion of the animation
+    u.setTimeScale(8)
+    doAfter(0.05) ->
+        u.setTimeScale(0.85) 
+    //Reset animation to stand after jumping
+    doAfter((distance/groundSpeed)) ->
```

Comment:
Spaces around binary operator, and you probably don't need the extra set of parenthesis.